sw/device/silicon_creator/lib/drivers/retention_sram.h

```diff
@@ -75,7 +91,14 @@ OT_ASSERT_MEMBER_OFFSET(retention_sram_t, owner, 2048);
 OT_ASSERT_SIZE(retention_sram_t, 4096);
 
 enum {
+  /**
+   * Engineering sample version.
+   */
   kRetentionSramVersion1 = 0x72f4eb2e,
+  /**
+   * Includes the `boot_svc_msg` field in the silicon creator area.
+   */
+  kRetentionSramVersion2 = 0x5b89bd6d,
```

Comment:
Just out of curiosity, are we planning to use version 1 anymore outside of the engineering sample?

Reply:
I don't think so.